snyk-monitor-deployment.yaml

```diff
@@ -66,8 +66,6 @@ spec:
             cpu: '1'
             memory: '2Gi'
         securityContext:
-          runAsUser: 10001
```

Comment:
is it "ok" to delete it also from here and the helm chart? what does it mean that we don't have it?

Reply:
It runs as the user and group specified in the container image (which are the same: https://github.com/snyk/kubernetes-monitor/blob/staging/Dockerfile#L36-L40) so we're good!